docs/2_auth.md

```diff
@@ -397,6 +398,24 @@ To use the provider, pass the following options:
 The default configuration allows everyone with Bitbucket account to authenticate. To restrict the access to the team members use additional configuration option: `--bitbucket-team=<Team name>`. To restrict the access to only these users who has access to one selected repository use `--bitbucket-repository=<Repository name>`.
 
 
+### Gitea Auth Provider
+
+1. Create a new application: `https://<< your gitea host >>/user/settings/applications`
+2. Under `Redirect URI` enter the correct URL i.e. `https://<proxied host>/oauth2/callback`
+3. Note the Client ID and Client Secret.
+4. Pass the following options to the proxy:
+
+```
+    --provider = "github"
```

Comment:
The format should either be `--provider="github"` (without the whitespace) or `--provider "github"`. In most of our documentation we have made use of the former, so maybe we should be consistant with this.